### PR TITLE
update readme with basic D3 and ES6 usage example

### PR DESCRIPTION
**Pull request by GeorgeStrakhov**
Added usage example with D3 to README, as per #64

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,12 +39,57 @@ I'm trying to keep it light so as not to slow down your render function. I want
 
 ## Installation
 
-You can install the package `react-faux-dom` from npm as you usually would. Then use webpack or browserify (etc) to bundle the source into your build. If you need a pre-built UMD version you can use [npmcdn][].
-
-You can find the latest version of the UMD version at https://npmcdn.com/react-faux-dom/dist/ReactFauxDOM.min.js
+You can install the package `react-faux-dom` from npm as you usually would. Then use webpack or browserify (etc) to bundle the source into your build. 
+If you need a pre-built UMD version you can use [npmcdn][]. You can find the latest version of the UMD version at https://npmcdn.com/react-faux-dom/dist/ReactFauxDOM.min.js
```
Comment (Olical) [inline README.md]: I think you need a blank line between these otherwise markdown joins them.

File: README.md [hunk 2]
```diff
@@ -39,12 +39,57 @@ I'm trying to keep it light so as not to slow down your render function. I want
...
+        </div>
+      )
+    }
+
+  });
+
+  export default MyReactClass;
+```
+
+### More usage info:
```
Comment (Olical) [inline README.md]: Probably don't need this header, I think the links trailing after the example is more than enough.

My only reservations with this example is that the class is usually referred to and imported as `ReactFauxDOM` for clarity. This example is also an example of mixins and animation, something that most people will not require initially.

Maybe a simpler example that does the same thing without `connectFauxDOM` and the mixins would be more concise? I really like the idea of an end to end example though :smile:

Comment (GeorgeStrakhov) [inline README.md]: IMHO for a relative newbie to both D3 and React (like myself) a self-explanatory end-to-end example that you can copy-paste and start playing with immediately is really what you need front and center in the README.

So I left mixins and animation on purpose there: assuming that people will need them sooner or later. I would may be add a comment to the `animateFauxDom` line stating that you will likely need this later and leaving a link to the example file...

But do let me know what you prefer and I'll implement your other comments and update the PR accordingly.

**Comment by Olical** [comment]
Sorry about the delay, distracted. Would you be able to set the syntax of that code block to JavaScript to enable highlighting. Also some of the indentation looks a little off.

The only thing I'm not so sure of is the fact that people will copy and paste this as a starting point, but that starting point happens to include mixins etc. The basic usage is far simpler, this is actually an example of advanced usage. Although I like it, I think it'd be best to have a vanilla example first then list this after as "and here's all the other things you can do".

**Pull request merged**